Raymond-Osy/Politico
# #163499784 continous integrations setup

#### What does this PR do?
It is an integration setup

#### Description of Task to be completed?
- add coveralls and nyc files to gitignore
- update the README.md file to contain badges and more app information
- install ci dependencies

#### How should this be manually tested?
- visit https://github.com/Raymond-Osy/Politico
- click on the badges on the readme file below to see app's report

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#163499784](https://www.pivotaltracker.com/n/projects/2239146)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
